web/public/mgmt/config.yml

```diff
@@ -1500,14 +1500,22 @@ collections:
           - label: Weight
             name: weight
             widget: number
-          - label: Task
+          - label: Task (this or License Task is required)
             name: task
             widget: "relation"
             collection: "tasks"
             search_fields: ["{{filename}}", "name", "urlSlug"]
             value_field: "{{filename}}"
             display_fields: ["{{filename}}"]
             options_length: 500
+          - label: License Task (this or Task is required)
```

Comment:
`
  name: validation
    label: Custom Validation
    widget: hidden
    required: true
    script: path/to/validation-script.js
`

[Dust] perhaps this is a larger discussion. In an instance like this, where one of two fields is required, but we are not making them both required, should we have a custom validation?

Reply:
Totally, we totally should, wasn't sure if that was possible in Netlify but looks like you answered that question, nice

Reply:
I'll look into it now

Reply:
So I played with this quite a bit and it seems like it should totally be possible but might be a bit of a pain. So after time boxing it I'm going to write a follow up chore for it.

Reply:
https://www.pivotaltracker.com/story/show/185532198